### PR TITLE
Feature/upgrade dependencies

### PR DESCRIPTION
**Pull request by tocker**
* Upgrade dependencies to fix "ua-parser-js" security issue.
* Implement dynamic test subject to allow tests to run on sources, (compiled) lib, and dist.
* Fix webpack configuration.

### CODE REVIEW

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1 @@
+pure-lockfile true
```
Comment (eitanp461) [inline .yarnrc]: Why?

File: test/ImageTest.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow, mount, render } from 'enzyme';
-import Image from '../src/components/Image';
-import Transformation from '../src/components/Transformation';
+// import {default as foo} from './cloudinary-proxy';
```
Comment (eitanp461) [inline test/ImageTest.js]: Delete

File: .yarnrc (hunk 1 above)
Comment (tocker) [reply .yarnrc]: Don't create a lock file, but allow the developer to use one if present.

File: test/ImageTest.js (hunk 2 above)
Comment (tocker) [reply test/ImageTest.js]: :-)

**Pull request merged**